unitTests/utils/ComponentsTypesTest.ts

```diff
@@ -5,7 +5,7 @@ import { MockEnvironmentBuilder } from '../MockEnvironment';
 
 export function ComponentsTypesTest() {
   describe('ComponentsTypes', () => {
-    const expectedFacets = ['Facet', 'FacetRange', 'FacetSlider', 'CategoryFacet', 'HierarchicalFacet'];
+    const expectedFacets = ['Facet', 'FacetRange', 'FacetSlider', 'CategoryFacet', 'HierarchicalFacet', 'TimespanFacet'];
```

Comment:
Should the SimpleFilter be included in this list?

Reply:
It's not in the `allFacetsType` so I guess not, although the DynamicFacet should be there